soccer/gameplay/tactics/stopped/circle_on_center.py

```diff
@@ -65,7 +65,10 @@ def goto_center(self):
         dirvec.rotate(robocup.Point(0, 0), -perRobot * ((num_robots - 1) / 2))
 
         for i in range(6):
-            pt = ball_pos + dirvec
+            pt = ball_pos + dirvec if num_robots > 0 else robocup.Point(
+                -constants.Field.Width, -constants.Field.Length)
+            if num_robots < 1:
+                pt = robocup.Point(-5, -5)
```

Comment:
So -5, -5 is just a really far away point. I guess I put that mostly for testing purposes, ideally it would be like (infinity, infinity)